integrations/spring.hbs.md

```diff
@@ -3,13 +3,30 @@
 This topic describes how to migrate Spring applications from Tanzu Application Service or
 Azure Spring Apps to Tanzu Application Platform.
 
-## <a id="prereqs"></a> Prerequisite
+## <a id="spring-cloud-config"></a> Migrating from Spring Cloud Config
 
-If your applications are currently using Spring Cloud Configuration Service or
-Spring Cloud Service Registry,
-[install the Application Configuration Service](../application-configuration-service/install-app-config-service.hbs.md)
+If your applications are currently using
```

Comment:
Looks like this change removed the prerequiste step for applications migrating from Spring Cloud Service registry to install ACS. Since we rely on ACS being installed in the documentation below, I don't think that's a good change.

Reply:
The installation of ACS is only a prerequisite for ACS capabilities. The one issue I see with the page is that it covers SCG, ACS and service to service communication topics. For SCG and service to service communication, ACS is not a prerequisite. I wonder if we need to have each section pulled out into its own page or maybe we can have each section mention what is needed to be installed?

Reply:
This PR is already merged so, if additional changes are needed, please raise a new PR.